.eslintrc

```diff
@@ -0,0 +1,41 @@
+{
+  "parser": "babel-eslint",
+
+  "parserOptions": {
+    "ecmaVersion": 6,
```

Comment:
Shame that eslint es version doesn't follow the new naming like `es2015` and `es2016`...